v2f/src/test/scala/org/broadinstitute/monster/etl/v2f/V2FUtilsSpec.scala

```diff
@@ -78,6 +82,9 @@ class V2FUtilsSpec extends PipelineSpec with Matchers {
     )
   }
 
+  private val dataDir =
+    (File.currentWorkingDirectory / "src/test/scala/org/broadinstitute/monster/etl/v2f").path.toAbsolutePath.toString
```

Comment:
There was a regression in Beam's handler for local files that caused relative paths w/ globs to stop working.